task5/src/RunningServletName.java

```diff
@@ -0,0 +1,19 @@
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class RunningServletName extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String name = req.getParameter("name");
+        if(name.length()<=100){
```

Comment:
Please observe the normative block structure:
```
if (name.length() <= 100) {
    // do something
} else {
    // something else
}
```
All spaces and braces must be placed this way